### ISSUE TITLE
Older browser versions querySelector error

### ISSUE THREAD

**Issue by hhatakeyama** [rpearce/react-medium-image-zoom#391]
## Issue Type

- [x] Bug report
- [ ] Feature request
- [ ] Question / support request
- [ ] Other

## Description

On older chrome versions (tested on v86.0) and samsung internet browser (tested on v13), the following error was reported on sentry:

`Failed to execute 'querySelector' on 'Element': ':is(img, svg, [role="img"], [data-zoom]):not([aria-hidden="true"])' is not a valid selector.`

![image](https://user-images.githubusercontent.com/5581539/212346935-b124ebff-b5c8-4678-8427-301cc6aa4941.png)

**Comment by rpearce** [comment]
Hi @hhatakeyama, and thank you for opening your issue.

Chrome  86 looks like it came out over 2 years ago, and the global usage of it is very small. For Samsung browser v13, it is even less.

Before I go about making any changes, I have a few questions:

* Can your Chrome browsers be updated?
* For Chrome, it looks like it might work if a flag is enabled by the user. Can they do this? `Support for :is() can be enabled with the Experimental Web Platform features flag`
* How critical is this tool for you? I am concerned that if you're supporting lots of old browsers, this may just be 1 of potentially a number of issues with using newer browser features. If it's blocking you, then could you have a conditional where you check for browser support for this `:is` and, if it's not supported, you only render a regular image?

Thanks for your time!

**Comment by rpearce** [comment]
@hhatakeyama Ok, I will look into this.

**Comment by rpearce** [comment]
I'm sorry I haven't circled back to this. Other things have taken priority. Is this still an issue for you, @hhatakeyama?

**Comment by rpearce** [comment]
@pacocoursey @hhatakeyama Is this what that would basically look like?

```js

let selector = `:is(img, svg, [role="img"], [data-zoom]):not([aria-hidden="true"])`

if (!CSS.supports(selector)) {
  selector = `img:not([aria-hidden="true"]), svg:not([aria-hidden="true"]), [data-zoom]:not([aria-hidden="true"])`
}
```

At which point maybe it's just better to do this...

```js
const selector = ['img', 'svg', '[data-zoom]']
  .map(x => `${x}:not([aria-hidden="true"])`)
  .join(',')

document.querySelector(selector)
```

What do y'all think?

**Comment by rpearce** [comment]
Can y'all please 🙏 try this fix out? `npm i react-medium-image-zoom@5.1.4-rc.0`
